src/Test/PdbUtilities/Reader/Token2SourceLineExporter.cs

```diff
@@ -404,7 +404,7 @@ internal IntHashTable(int capacity, int loadFactorPerc)
             {
                 if (capacity < 0)
                     throw new ArgumentOutOfRangeException(nameof(capacity), "ArgumentOutOfRange_NeedNonNegNum");
-                if (!(loadFactorPerc >= 10 && loadFactorPerc <= 100))
+                if (loadFactorPerc is not (>= 10 and <= 100))
```

Comment:
open question on if we could push the 'not' through as long as it didn't produce more nots.  in this case, it would flip values and change and->or, but the total complexity of the expression would go down.

Reply:
Same thing applies to some of the cases where the refactoring introduces lots of `and not`'s to the expression.

Perhaps a second fixer option of "Use `is not` (and invert check)" if the fixer would need to add parens, or more than two `not`s or something.